static/js/apps/explore/app.tsx

```diff
@@ -365,14 +370,16 @@ const fetchDetectAndFufillData = async (
   dc: string,
   disableExploreMore: string,
   detector: string,
-  llmApi: string
+  llmApi: string,
+  testMode: string
 ) => {
   const detectorTypeArg = detector ? `&detector=${detector}` : "";
   const llmApiArg = llmApi ? `&llm_api=${llmApi}` : "";
+  const testModeArg = testMode ? `&test=${testMode}` : "";
   try {
     const startTime = window.performance ? window.performance.now() : undefined;
     const resp = await axios.post(
-      `/api/explore/detect-and-fulfill?q=${query}${detectorTypeArg}${llmApiArg}`,
+      `/api/explore/detect-and-fulfill?q=${query}${detectorTypeArg}${llmApiArg}${testModeArg}`,
```

Comment:
for later, shall we move these from url param to POST data? There is a combination of both now.

Reply:
That makes sense!